releases/release-1.35/release-notes/release-notes-draft.json

```diff
@@ -5694,6 +5694,115 @@
+    "text": "Kubernetes is now built using Go 1.25.5",
+    "markdown": "Kubernetes is now built using Go 1.25.5 ([#135609](https://github.com/kubernetes/kubernetes/pull/135609), [@cpanato](https://github.com/cpanato)) [SIG Release and Testing]",
+    "author": "cpanato",
+    "author_url": "https://github.com/cpanato",
+    "pr_url": "https://github.com/kubernetes/kubernetes/pull/135609",
+    "pr_number": 135609,
+    "areas": [
+      "test",
+      "release-eng"
+    ],
+    "kinds": [
+      "feature"
+    ],
+    "sigs": [
+      "release",
+      "testing"
+    ],
+    "feature": true,
+    "is_mapped": true,
     "duplicate": true
+  },
+  "135271": {
+    "commit": "03e14cc9432975dec161de1e52d7010f9711a913",
+    "text": "Etcd: Update etcd to v3.6.6",
+    "markdown": "Etcd: Update etcd to v3.6.6 ([#135271](https://github.com/kubernetes/kubernetes/pull/135271), [@bzsuni](https://github.com/bzsuni) [SIG API Machinery, Cloud Provider, Cluster Lifecycle, Etcd and Testing]",
+    "author": "bzsuno",
+    "author_url": "https://github.com/bzsuni",
+    "pr_url": "https://github.com/kubernetes/kubernetes/pull/135271",
+    "pr_number": 135271,
+    "areas": [
+      "provider/gcp",
+      "test"
+    ],
+    "kinds": [
+      "cleanup"
+    ],
+    "sigs": [
+      "api-machinery",
+      "cloud-provider",
+      "cluster-lifecycle",
+      "etcd",
+      "testing"
+    ],
+    "duplicate": true,
+    "is_mapped": true
+  },
+  "135560": {
+    "commit": "615ac0d2996f36fe4df7410e1168a8577e7045ba",
+    "text": "Fixes a bug where MutatingAdmissionPolicy would fail to apply to objects with duplicate list items (like env vars).",
+    "markdown": "Fixes a bug where MutatingAdmissionPolicy would fail to apply to objects with duplicate list items (like env vars). ([#135560](https://github.com/kubernetes/kubernetes/pull/135560), [@lalitc375](https://github.com/lalitc375) [SIG API Machinery]",
+    "author": "lalitc375",
+    "author_url": "https://github.com/lalitc375",
+    "pr_url": "https://github.com/kubernetes/kubernetes/pull/135560",
+    "pr_number": 135560,
+    "areas": [
+      "apiserver"
+    ],
+    "kinds": [
+      "bug"
+    ],
+    "sigs": [
+      "api-machinery"
+    ],
+    "duplicate": true,
+    "is_mapped": true
+  },
+  "135499": {
+    "commit": "2711e8cb53987ad7aca9eee21e1be3a4961fea39",
+    "text": "Fix a bug in the kube-apiserver where a malformed Service without name can cause high CPU usage. The bug is present on the new Cluster IP allocators enabled with the feature MultiCIDRServiceAllocator (enabled by default since 1.33).",
+    "markdown": "Fix a bug in the kube-apiserver where a malformed Service without name can cause high CPU usage. The bug is present on the new Cluster IP allocators enabled with the feature MultiCIDRServiceAllocator (enabled by default since 1.33). ([#135499](https://github.com/kubernetes/kubernetes/pull/135499), [@aojea](https://github.com/aojea) [SIG Testing]",
```

Comment:
```suggestion
    "markdown": "Fixes a bug in the kube-apiserver where a malformed Service without name can cause high CPU usage. The bug is present on the new Cluster IP allocators enabled with the feature `MultiCIDRServiceAllocator` (enabled by default since `1.33`). ([#135499](https://github.com/kubernetes/kubernetes/pull/135499), [@aojea](https://github.com/aojea) [SIG Testing]",
```